src/java.base/share/native/libjli/java.c

```diff
@@ -1921,14 +1921,19 @@ DescribeModule(JNIEnv *env, char *optString)
  * Prints default usage or the Xusage message, see sun.launcher.LauncherHelper.java
  */
 static void
-PrintUsage(JNIEnv* env, jboolean doXUsage)
+PrintUsage(JNIEnv* env, jboolean doXUsage, jboolean conciseUsage)
```

Comment:
This would be cleaner to replace the booleans with an enum (concise, X, full).
Since conciseUsage overrides doXusage, then it should preceed Xusage in the argument list.

Reply:
Thanks. I've attempt to do that here:
https://github.com/openjdk/jdk/pull/21411/commits/1fe578185e5c8f575dabef5c5fb869e93f218c72